### ISSUE TITLE
data

### ISSUE THREAD

**Issue by Torrie-ll** [Zian-Xu/Swin-MAE#12]
Hello, I am a graduate student. Your article is very helpful for my research and I am very interested. Can you send me a dataset for my research?

**Comment by Zian-Xu** [comment]
Of course, please leave your contact information, either email or WeChat.

**Comment by Torrie-ll** [comment]
Thank you very much for your help in my research!
My email is: ***@***.***
My WeChat account is: Torrie1128_Tl





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Zian-Xu/Swin-MAE"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月9日(星期四) 晚上9:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Zian-Xu/Swin-MAE] data (Issue #12)





 
Of course, please leave your contact information, either email or WeChat.
